controllers/src/main/scala/cromwell/pipeline/controller/ControllerModule.scala

```diff
@@ -9,4 +9,5 @@ class ControllerModule(serviceModule: ServiceModule)(implicit executionContext:
   lazy val userController: UserController = new UserController(serviceModule.userService)
   lazy val projectController: ProjectController = new ProjectController(serviceModule.projectService)
   lazy val projectFileController: ProjectFileController = new ProjectFileController(serviceModule.projectFileService)
+  lazy val inputNodeController: InputNodeController = new InputNodeController(serviceModule.inputNodeService)
```

Comment:
Think about naming. "InputNode"!?

Reply:
Yep, I agree about bad naming, will think about it. Mb you can help me :)